src/HttpClientInterception/HttpRequestInterceptionBuilder.cs

```diff
@@ -132,25 +132,7 @@ public HttpRequestInterceptionBuilder For(params Predicate<HttpRequestMessage>[]
     /// </remarks>
     public HttpRequestInterceptionBuilder For(params Func<HttpRequestMessage, Task<bool>>[] predicates)
```

Comment:
Still seems like this won't do the expected thing if you called it with an empty array.